homeassistant/components/ios/__init__.py

```diff
@@ -230,9 +230,6 @@ async def async_setup(hass, config):
 
     hass.data[DOMAIN] = ios_config
 
-    # No entry support for notify component yet
-    discovery.load_platform(hass, "notify", DOMAIN, {}, config)
```

Comment:
This doens't just set up Notify. This also loads the iOS platfor for notify.

Reply:
This call requires the `config` to not be empty (falsy). Should we add a check before doing this call?

We should really add config entries support to notify integration, but that's a bigger architecture problem.